wallet/src/Pos/Wallet/Web/Methods/Misc.hs

```diff
@@ -155,16 +156,16 @@ syncProgress = do
 -- NTP (Network Time Protocol) based time difference
 ----------------------------------------------------------------------------
 
-localTimeDifference :: (NtpCheckMonad m, MonadMockable m) => m Word
+localTimeDifference :: (NtpCheckMonad m, MonadMockable m) => m Integer
 localTimeDifference =
     diff <$> getNtpStatusOnce
   where
-    diff :: NtpStatus -> Word
+    diff :: NtpStatus -> Integer
     diff = \case
         NtpSyncOk -> 0
         -- `NtpSyncOk` considered already a `timeDifferenceWarnThreshold`
         -- so that we can return 0 here to show there is no difference in time
-        NtpDesync diff' -> fromIntegral diff'
+        NtpDesync diff' -> toMicroseconds diff'
```

Comment:
Probably just a `fromIntegral` like before would have suffice, but this one is slightly more self-explanatory.